src/styles/components/buttons/_buttons.scss

```diff
@@ -12,7 +12,7 @@
   line-height: 2;
   letter-spacing: 0.12em;
   text-transform: uppercase;
-  border-radius: $default-radius;
+  border-radius: var(--mc-scale-3);
```

Comment:
Because this change only affects buttons, we don't want the default radius changing for the tiles / other rounded components, I advised Will to change to the scale here.